assets/src/app/ato/Products/Static.tsx

```diff
@@ -76,13 +79,13 @@ const ProductInfo: {[Identifier: string]: ProductTemplate } = {
             'SCAP': require('@app/assets/markdown/products/openstack-platform-13/SCAP.md').default,
         }
     },
-    'rhel-7': {
+    'enterprise-linux-7': {
```

Comment:
This change breaks the ui. Making `Overview` and `SCAP` sub-pages unavailable.

Not sure what we are trying to achieve here, but we need significant amount of changes to rename product to something else while keeping the ui working.